check_sensitive_keywords.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Check if the environment variable GIT_ENV_VAR_S3_IBM_TXT_URL is set
+if [ -z "${GIT_ENV_VAR_S3_IBM_TXT_URL+ x}" ]; then
```

Comment:
This is an environment variable the github owner needs to define on github settings.